app/views/organizations/show.html.erb

```diff
@@ -1,4 +1,5 @@
 <div class="org-form">
+  <%= link_to "Edit Organization", edit_organization_path(@organization) %>
```

Comment:
Should it be a button or just a link?

I imagine if you wanted a button you could just use the submit stylings that are already on the page like this:
<%= link_to "Edit Organization", edit_organization_path(@organization), html_options = {type: "submit"} %>

Reply:
I like a button 👍  thanks for the suggestion!